README.md

```diff
@@ -2,10 +2,20 @@
 dev: ![CI](https://github.com/hpi-swa-teaching/OhmPrettyPrint/workflows/CI/badge.svg?branch=dev)  
 master: ![CI](https://github.com/hpi-swa-teaching/OhmPrettyPrint/workflows/CI/badge.svg?branch=master)
 
+OmegaPrint is a tool which prettifies your code in a consistent way. It uses [Ohm/S](https://github.com/hpi-swa/Ohm-S) as a parser.
+
 ## How to install
 ```smalltalk
 Metacello new
-	baseline: 'OhmPrettyPrint';
-	repository: 'github://HPI-SWA-Teaching/OhmPrettyPrint:dev/packages';
+	baseline: 'OmegaPrint';
+	repository: 'github://HPI-SWA-Teaching/OmegaPrint:dev/packages';
 	load
 ```
+
+After installing OmegaPrint you will find a button in all your code holder. For example the browser. 
+Clicking on it OmegaPrint will show you a prettified version of your code.
+
+##Comments
+OmegaPrint does basic comment-formatting.
+There is a problem because comments are not in the CST:
+Therefore, we use the method OhmNode >> skippedSpacesNodes which gives us Nodes containing all spaces and comments, which are before the Node on which we call the method.
```

Comment:
Comment-formatting is more complicated than code-formatting. This happens due to the fact, that comments and whitespace is not in the CST. Therefore...